dev/TabView/TabView.cpp

```diff
@@ -406,7 +394,13 @@ void TabView::UpdateTabContent()
         }
         else
         {
-            if (auto container = ContainerFromItem(SelectedItem()).as<winrt::ListViewItem>())
+            auto tvi = SelectedItem().try_as<winrt::TabViewItem>();
+            if (!tvi)
```

Comment:
I think you can just use ContainerFromItem() here without this two-step check - the item should return itself if it itself is a container.

Reply:
I thought so too, and I truly wish that I could because I agree that it absolutely should.... and yet it doesn't. There is some issue when doing a drag/drop where for a little while ContainerFromItem() returns null even though the SelectedItem() IS A CONTAINER. Gnashing of teeth and pulling of hair occurred, followed by this hack.

Reply:
Really, huh.  Weird.  Well, carry on then!